### PR TITLE
Update read me and add .envTemplate

### PR DESCRIPTION
**Pull request by thomasyu888**
- You cannot use environmental variables for docker-compose.  Andrew tried and it didn't work.  Also `docker-compose up` does not have `-e` capability

- Edit TOIL params readme

- Add in readme for setting up workflow entity on Synapse and linking to evaluation Id

- Add in `.envTemplate`

### CODE REVIEW

**Comment by thomasyu888** [comment]
@brucehoff  Please review

File: README.md [hunk 1]
```diff
@@ -40,9 +40,23 @@ TODO:  Automatically lookup ROOT_TEMPLATE in Dockstore
 
 Will print out created Project ID and the value for the `EVALUATION_TEMPLATES` in the following step.
 
+#### Linking your workflow with evaluation queue manually
+
+If you already have an existing project and do not want to follow the `To use` instructions above, here are instructions on how to link your workflow with an Evaluation queue.
```
Comment (brucehoff) [inline README.md]: `To use` -> `Create a project ...`

File: README.md [hunk 2]
```diff
@@ -72,7 +86,7 @@ When using `DOCKER_CERT_PATH_HOST` you must also add the following under `volume
 ```
 {"9614045":"syn16799953"}
 ```
-- `TOIL_CLI_OPTIONS` - (optional) Space separated list of options.  See https://toil.readthedocs.io/en/3.15.0/running/cliOptions.html.  Example:
+- `TOIL_CLI_OPTIONS` - (optional, but highly recommended) Space separated list of options. (Without the toil parameters, you may run into errors with running the workflow hook).  See https://toil.readthedocs.io/en/3.15.0/running/cliOptions.html.  Example:
```
Comment (brucehoff) [inline README.md]: "with running the workflow hook" 
change to
"when a new workflow job is started"

**Comment by brucehoff** [comment]
Thanks for the update.

**Pull request merged**